docs/topics/proc_starting-your-application-locally-in-debugging-mode.adoc

```diff
@@ -1,19 +1,20 @@
+// This is a parameterized module. Parameters used:
+//
+//   parameter-debug-property: The Java application property used to specify the debug port
+//   parameter-maven-goal: Name of the runtime and the goal for executing debugging, e. g. "vertx:run"
+//
+// Rationale: This procedure is identical in Swarm and Vert.x
+
+
 [#starting-your-application-locally-in-debugging-mode_{context}]
 = Starting Your Application Locally in Debugging Mode
 
 One of the ways of debugging a Maven-based project is manually launching the application while specifying a debugging port, and subsequently connecting a remote debugger to that port.
 This method is applicable at least to the following deployments of the application:
 
+ifdef::vert-x[* When launching the application manually using the `mvn {parameter-maven-goal}` goal. This starts the application with debugging enabled.]
 ifdef::wf-swarm[]
-* When launching the application manually using the `mvn wildfly-swarm:run`goal.
-endif::[]
-ifdef::vertx[]
-* When launching the application manually using the `mvn vertx:debug` goal. This starts the application with debugging enabled.
-endif::[]
-ifdef::spring-boot[]
-* When launching the application manually using the `mvn spring-boot:run`goal.
-endif::[]
-ifdef::wf-swarm[]
+* When launching the application manually using the `mvn {parameter-maven-goal}` goal.
 * When starting the application without waiting for it to exit using the `mvn wildfly-swarm:start` goal.
 This is useful especially when performing integration testing.
 endif::[]
```

Comment:
The line below this one says "When using the Arquillian adapter for {runtime}.", but there's no Arquillian adapter for Vert.x.